Passage à Debian 12

philippemilink opened the pull request:
Préparation du passage à Debian 12.

Surtout pour voir si la CI passe pour l'instant.

philippemilink committed:
Passage à Debian 12

philippemilink committed:
Indexe dans ElasticSearch les fixtures chargées

philippemilink committed:
Met à jour les dépendances Ruby

StaloneLab commented:
Salut, généralement je ne fais pas d'objections sur ce dépôt, mais là il y a deux choses qui retiennent mon attention en ce qui concerne ZMarkdown :

- j'ai corrigé récemment (et repris aujourd'hui) des problèmes pour Node 18 et 20 dans ZMarkdown. En l'occurence il s'agit de changements internes à Node.js, qui affectent essentiellement la gestion des erreurs (y compris sur la dernière version publiée, qui date de juillet). Je ne suis pas certain de l'impact que ça peut avoir en production, donc je vois deux solutions : soit tester que ça ne nous pose pas de problème (je m'occuper de créer des contenus potentiellement problématiques), soit mettre à jour ZMd, mais la nouvelle version ne sortira probablement pas de suite ;
- les tableaux ne fonctionnent plus sur les versions récentes de LaTeX (chez moi TeXLive 2023), et je ne sais pas à partir de quelle version de TeXLive. Un correctif est en cours, mais si le but est de déployer D12 rapidement il faudrait s'assurer qu'on sait exporter des tableaux (un tableau simple suffit normalement pour vérifier si ça fonctionne).

philippemilink committed:
Installe certbot

philippemilink committed:
Réorganise roles/common/tasks/main.yml

philippemilink committed:
Configure /etc/ssh/sshd_config

philippemilink committed:
Installe service-zds.sh dans /root/bin

philippemilink committed:
Ajoute un rôle backup (surtout pour la bêta)

philippemilink committed:
Supprime une ligne vide inutile

philippemilink committed:
Améliore le rôle backup

philippemilink committed:
Crée les dossiers pour les dépôts Borg

philippemilink committed:
Change les paramètres pour activer HTTPS

Permet de définir une configuration avec HTTPS sans avoir encore
de certificat.

philippemilink committed:
Installe le paquet cron

philippemilink committed:
Ajoute un rôle pour installer zds-antispam

philippemilink committed:
Configure la timezone

philippemilink committed:
Corrige le chemin de ssl_dhparam dans nginx/snippets/ssl.conf

philippemilink committed:
Ajustements dans le script de restauration des sauvegardes

philippemilink committed:
Met à jour le nom de domaine du serveur de bêta

philippemilink committed:
Corrige une typo
